lib/email-validation/blacklisted_email.rb

```diff
@@ -16,19 +12,13 @@ class BlacklistedEmail < ::ActiveRecord::Base
 
     before_save :parse_email
 
-    workflow do
-      state :bounce
-      state :unsubscribe
-      state :admin
-    end
-
     ORIGIN_ADMIN = 'admin'
```

Comment:
So why do we have them here?;) I haven't changed that line;) But seriously I use it in the bounce_or_admin? method and if you call "git grep ORIGIN_ADMIN" on MV repo, you'll find it's used there as well.